eq-publisher/src/middleware/fetchData.js

```diff
@@ -1,5 +1,9 @@
 const { get } = require("lodash");
 
+const setQuestionnaire = async (res, questionnaire) => {
```

Comment:
Is this async needed?

Reply:
Apparently not, eslint satisfied without it 👍